src/main/groovy/io/repaint/maven/tiles/TileValidator.groovy

```diff
@@ -112,6 +112,11 @@ class TileValidator {
 			validModel = null
 		}
 
+		if (model.build?.extensions) {
+			log.error("Tile has extensions and must not have")
+			validModel = null
+		}
+
```

Comment:
We probably also want to check for `<extensions>true</extensions>` in the general plugin declaration as well ( that's mostly where I've encountered adding extensions personally ).

Reply:
Good point. Done.